datahub-web-react/src/app/entity/shared/containers/profile/sidebar/DataProduct/DataProductSection.tsx

```diff
@@ -0,0 +1,103 @@
+import { EditOutlined } from '@ant-design/icons';
+import { Button, Modal, Typography, message } from 'antd';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { SidebarHeader } from '../SidebarHeader';
+import { useEntityData } from '../../../../EntityContext';
+import { EMPTY_MESSAGES } from '../../../../constants';
+import SetDataProductModal from './SetDataProductModal';
+import { DataProductLink } from '../../../../../../shared/tags/DataProductLink';
+import { useBatchSetDataProductMutation } from '../../../../../../../graphql/dataProduct.generated';
+import { DataProduct } from '../../../../../../../types.generated';
+
+const EmptyText = styled(Typography.Paragraph)`
+    &&& {
+        border-top: none;
+        padding-top: 0;
+    }
+`;
+
+interface Props {
+    readOnly?: boolean;
+}
+
+export default function DataProductSection({ readOnly }: Props) {
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    const { entityData, urn } = useEntityData();
+    const [batchSetDataProductMutation] = useBatchSetDataProductMutation();
+    const [dataProduct, setDataProduct] = useState<DataProduct | null>(null);
+    const dataProductRelationships = entityData?.dataProduct?.relationships;
+    const siblingUrns: string[] = entityData?.siblings?.siblings?.map((sibling) => sibling?.urn || '') || [];
+
+    useEffect(() => {
+        if (dataProductRelationships && dataProductRelationships.length > 0) {
+            setDataProduct(dataProductRelationships[0].entity as DataProduct);
+        }
+    }, [dataProductRelationships]);
+
+    function removeDataProduct() {
+        batchSetDataProductMutation({ variables: { input: { resourceUrns: [urn, ...siblingUrns] } } })
+            .then(() => {
+                message.success({ content: 'Removed Data Product.', duration: 2 });
+                setDataProduct(null);
+            })
+            .catch((e: unknown) => {
+                message.destroy();
+                if (e instanceof Error) {
+                    message.error({ content: `Failed to remove data product: \n ${e.message || ''}`, duration: 3 });
```

Comment:
Recently ive been hiding the raw error message and just writing something like:

Failed to remove Data Product. An unknown error occurred.